src/OrchardCore.Modules/OrchardCore.Email/Extensions/OrchardCoreBuilderExtensions.cs

```diff
@@ -0,0 +1,21 @@
+using Microsoft.Extensions.Configuration;
+using OrchardCore.Environment.Shell.Configuration;
+using OrchardCore.Email;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    public static class OrchardCoreBuilderExtensions
+    {
+        public static OrchardCoreBuilder ConfigureEmailSettings(this OrchardCoreBuilder builder)
```

Comment:
Add a note about this to [this section](https://docs.orchardcore.net/en/dev/docs/reference/core/Configuration/#ioptions-configuration) since something like this is the current sample. The sample can remain, just add something along the lines of "Full such configuration for `SmtpSettings` is available via the `ConfigureEmailSettings` extension method, see [link to docs page]."